array_schema.go

```diff
@@ -425,3 +426,22 @@ func (a *ArraySchema) Type() (ArrayType, error) {
 
 	return ArrayType(arrayType), nil
 }
+
+// Handle a LoadArraySchemaRequest. This is used by TileDB-Cloud
+func HandleLoadArraySchemaRequest(array *Array, serializationType SerializationType, request *Buffer) (*Buffer, error) {
+	response, err := NewBuffer(array.context)
+	if err != nil {
+		return nil, fmt.Errorf("error deserializing load array schema request: %s", array.context.LastError())
```

Comment:
milsleading, there is no deserialization here. Use sth like `cannot allocate response buffer`